bootstrap-project/build.gradle

```diff
@@ -0,0 +1,131 @@
+        classpath "org.jetbrains.kotlin:kotlin-allopen:$kotlin_version"
+        classpath 'com.github.jengelman.gradle.plugins:shadow:5.1.0'
+    }
+}
+
+plugins {
+    id "org.jetbrains.kotlin.plugin.spring" version "1.3.21"
+    id 'jp.co.soramitsu.sora-plugin' version '0.1.2'
+}
+
+apply plugin: 'application'
+apply plugin: 'com.github.johnrengelman.shadow'
+shadowDistZip.classifier = 'shadow'
+shadowDistTar.classifier = 'shadow'
+
+group 'd3'
+mainClassName = "jp.co.soramitsu.bootstrap.BootstrapMain"
+
+// name of jar file that the shadowJar plugin generates
+shadowJar.archiveName = 'app.jar'
+
+// sora-plugin configs
+soramitsu {
+    projectGroup = 'd3-deploy'
+    docker {
+        // docker tag
+        tag = System.getenv("TAG")
+        // jar file that is used in the generated Dockerfile
+        jar = new File("build/libs/app.jar")
+        // the rest in configured using env variables
+    }
+}
+
+repositories {
+    mavenCentral()
+    maven { url 'https://jitpack.io' }
+}
+
+dependencies {
+    implementation("com.github.d3ledger.notary:notary-commons:$notary_version")
+    implementation("com.github.d3ledger.d3-eth:eth:$d3EthVersion")
+    implementation("com.github.soramitsu.chain-adapter:chain-adapter-client:$chain_adapter_client_version")
+
+    // kotlin related
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlinVersion"
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.0.1'
+    testImplementation "org.jetbrains.kotlin:kotlin-test:$kotlinVersion"
+
+    //Iroha-java related
+    implementation("com.github.warchant:ed25519-sha3-java:2.0.1")
+    implementation("com.github.hyperledger.iroha-java:client:$iroha_java_version")
+    implementation("com.github.hyperledger.iroha-java:testcontainers:$iroha_java_version")
+
+    //Spring-boot
+        implementation("org.springframework.boot:spring-boot-starter:$springBootVersion") {
+            exclude group: 'org.springframework.boot', module: 'spring-boot-starter-logging'
+        }
+        implementation("org.springframework.boot:spring-boot-starter-web:$springBootVersion") {
+            exclude group: 'org.springframework.boot', module: 'spring-boot-starter-tomcat'
+            exclude group: 'org.springframework.boot', module: 'spring-boot-starter-logging'
+            exclude group: 'org.hibernate'
+        }
+        implementation "io.springfox:springfox-swagger2:$springfoxSwagger2Version"
+        implementation "io.springfox:springfox-swagger-ui:$springfoxSwagger2Version"
+
+    // Changelog interface
+    implementation("com.github.d3ledger.bootstrap-changelog:changelog-interface:$changelog_version")
+
+    //TODO this is very dirty hack. This dependency must be removed after migration to newer version of Spring Boot
+    // https://mvnrepository.com/artifact/org.hibernate/hibernate-validator
+//        implementation 'org.hibernate:hibernate-validator:6.0.14.Final'
```

Comment:
is not needed anymore? why not to remove?

Reply:
solved